test/e2e/framework.go

```diff
@@ -183,6 +185,31 @@ type PodIPs struct {
 	ipStrings []string
 }
 
+type ConfigMap struct {
+	APIVersion string
+	Data       map[string]string `yaml:"data"`
+}
+
+func GetAntreaConfig() (ConfigMap, error) {
+	configfile := os.Getenv("YAML_FILE")
```

Comment:
I don't think it is a good idea to use environment var here. Please change it to an option as well.

Reply:
I am not sure if it is ok to get feature gate by reading the antrea yml file. What if some testcase changes some feature gate? I think k8s api should be a better choice. Is there another way to make `skipIfxxx` because `setupTest`?

Reply:
I will change it to pass via an option, for your second comment, I don't think we can use k8s api because setupTest will do antrea deployment. if we want to check feature gates info before setupTest,  it means there is no antrea-configmap yet.  if testcase change the feature gates, I think it will impact the antrea-config in the k8s env only.  Is there any test case to call another top e2e file?